java/src/main/java/com/google/cloud/dataproc/templates/mongo/README.md

```diff
@@ -0,0 +1,45 @@
+# Mongo To BigQuery
```

Comment:
Merge this content with java/src/main/java/com/google/cloud/dataproc/templates/databases/README.md